crates/network-scanner/src/mdns.rs

```diff
@@ -19,10 +19,34 @@ impl MdnsDaemon {
     pub fn get_service_daemon(&self) -> mdns_sd::ServiceDaemon {
         self.service_daemon.clone()
     }
+
+    pub fn stop(&self) {
+        let receiver = match self.service_daemon.shutdown() {
+            Ok(receiver) => receiver,
+            Err(e) => {
+                // if e is try again, we should try again, but only once
+                let result = if matches!(e, mdns_sd::Error::Again) {
+                    self.service_daemon.shutdown()
+                } else {
+                    tracing::error!(error = %e, "Failed to shutdown service daemon");
```

Comment:
The macro is in scope
```suggestion
                    error!(error = %e, "Failed to shutdown service daemon");
```